### DISCUSSION TITLE
[Feature Request] Admin can control number of versions accessible via versions dropdown.

### DISCUSSION THREAD

**Discussion by peytonchance** [craftcms/cms#8609] [Ideas] [answered]
### Description
Currently the number of versions an admin can access is limited to 10 by default. The request here is to add control via settings or config to increase (or limit) the number of versions are accessible via the dropdown.

![image](https://user-images.githubusercontent.com/23386632/49658936-2069e900-fa11-11e8-85a1-f6d073d5c2bb.png)

https://github.com/craftcms/cms/blob/develop/src/templates/entries/_revisions.html#L67

`{% set versions = craft.app.entryRevisions.getVersionsByEntryId(entry.id, entry.siteId, 10) %}`

**Comment by mmikkel** [comment]
This is still a thing as of Craft 3.7. It's a bit odd that the default `maxRevisions` setting is `50`, when no more than the latest 10 will ever be easily recoverable or available to authors 🤷‍♂️ 

It'd be great if the revisions dropdown made all revisions available, perhaps in a paginated list (or with infinite scrolling) inside the dropdown to avoid performance issues.

**Comment by andreimoment** [comment]
I agree that the dropdown interface should allow access to the max number of revisions specified in the config.

Please remove the hard-coded limit in the dropdown or provide another way to access the full list.

Our editorial team goes over 40+ revisions in rapid succession and sometimes needs to view/revert an entry 20 or more revisions back. They get very nervous when an important revision starts slipping out of reach.

**Comment by darylknight** [comment]
I'd like this added too. I've just had a client ask for permission to see more than 10 revisions. I knew they were stored in the database, so when we built the site I told them that Craft keeps a revision history of all the times they've saved it - which it does... they just can't get to them.
I find it very strange that if you store 50 revisions in the database, it's impossible for the client to get to those through the control panel. Please consider adding something that allows clients to go back through all the revisions that are stored in the database for that entry.

**Comment by brandonkelly** [comment, marked as answer]
This is now resolved for Craft 4.4.

Entries which have more than 10 revisions will start showing a “View all revisions” item at the bottom of their revision menu, which links off to a revision index page for the entry, which paginates through all of the entry’s revisions. (#12211)

**Reply by brandonkelly** [reply]
Craft 4.4 is out with that change!

**Comment by darylknight** [comment]
This is amazing, thank you!
